src/Pode.psd1

```diff
@@ -478,5 +481,7 @@
             ReleaseNotes = 'https://github.com/Badgerati/Pode/releases/tag/v$version$'
 
         }
+        PwshVersionsUntested = '$versionUntested$'
+        PwshVersionSupported = '$versionSupported$'
```

Comment:
Probably worth pluralising this, so it's `PwshVersionsSupported` to match the other.

Wondering if this might look nicer as a hashtable: 🤔 
```powershell
PwshVersions = @{
    Untested  = '$versionUntested$'
    Supported = '$versionSupported$'
}
```

Reply:
I agree 
I'll change tomorrow